src/test/java/Homework17b.java

```diff
@@ -0,0 +1,38 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+
+public class Homework17b extends BaseTest{
+
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+        login("demo@class.com", "te$t$tudent");
+        login("aidamtc@gmail.com", "te$t$tudent");
+        Thread.sleep(2000);
+        grabASong();
+
+        Actions acts = new Actions(driver);
+        acts.clickAndHold(song)
+                .release(playlist)
+                .build()
+                .perform();
+
+        playlist.click();
+        acts.dragAndDrop(song, playlist).build().perform();
```

Comment:
Where this variables are comming from:
![A31-QA-Automation-Framework – Homework17b java  A31-QA-Automation-Framework test](https://user-images.githubusercontent.com/49252454/211911868-da08f5f1-4e7a-4cc8-9439-daddd2461244.jpg)

It looks like somthing went wrong with commits
Try to fix it